lmnet/tox.ini

```diff
@@ -8,7 +8,7 @@ skipsdist = True
 recreate = True
 deps =
     -r{toxinidir}/../gpu.requirements.txt
```

Comment:
Oh, tests by tox was failed...
We also need to add deps of `pycocotools==2.0.0` here. 😄

Reply:
of course. thanks.